wp-cli-rename-db-prefix.php

```diff
@@ -213,21 +213,28 @@ protected function update_blog_options_tables() {
 		// todo should this really go after update_options_table, and reuse the same query?
 		// todo is this running on the root site twice b/c update_options_table() hits that too? should call either that or this, based on is_multisite() ?
 
-    	$sites = wp_get_sites( array( 'limit' => false ) );   //todo can't use b/c already renamed tables?
+		//Check Wordpress Version > 4.6
+		if ( function_exists( 'wp_get_sites' ) ) {
```

Comment:
This line looks like it might be backwards, won't `wp_get_sites()` always exist, since it's the older function?

I don't think we need to support both, though; it's fine to only support `4.6+`, and removing the extra code would make things simpler.